src/main/java/uk/gov/hmcts/reform/finrem/documentgenerator/controller/DocumentConversionController.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.finrem.documentgenerator.controller;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.finrem.documentgenerator.model.Document;
+import uk.gov.hmcts.reform.finrem.documentgenerator.service.DocumentConversionService;
+import uk.gov.hmcts.reform.finrem.documentgenerator.service.DocumentManagementService;
+
+@RestController
+@RequiredArgsConstructor
+@Api(value = "Document conversion", tags = {"Document conversion"})
+@Slf4j
+public class DocumentConversionController {
+
+    private final DocumentConversionService documentConversionService;
+    private final DocumentManagementService documentManagementService;
+
+    @ApiOperation(value = "Deletes document from the evidence management.", tags = {"Document Generation"})
```

Comment:
Is it actually deleting document from Evidence Management? Looks like it's storing a PDF version